Tests/Block/BlockContextManagerTest.php

```diff
@@ -114,15 +114,15 @@ public function testWithInvalidSettings()
 //     */
 //    public function testGetWithException()
 //    {
-//        $service = $this->getMock('Sonata\BlockBundle\Block\BlockServiceInterface');
+//        $service = $this->createMock('Sonata\BlockBundle\Block\BlockServiceInterface');
 //        $service->expects($this->exactly(2))->method('setDefaultSettings');
 //
-//        $blockLoader = $this->getMock('Sonata\BlockBundle\Block\BlockLoaderInterface');
+//        $blockLoader = $this->createMock('Sonata\BlockBundle\Block\BlockLoaderInterface');
 //
-//        $serviceManager = $this->getMock('Sonata\BlockBundle\Block\BlockServiceManagerInterface');
+//        $serviceManager = $this->createMock('Sonata\BlockBundle\Block\BlockServiceManagerInterface');
 //        $serviceManager->expects($this->exactly(2))->method('get')->will($this->returnValue($service));
 //
-//        $block = $this->getMock('Sonata\BlockBundle\Model\BlockInterface');
+//        $block = $this->createMock('Sonata\BlockBundle\Model\BlockInterface');
```

Comment:
Was you can see, I didn't comment this method I just fixed the call. Also we could remove the dead code...